pkg/repo/repo.go

```diff
@@ -55,6 +55,12 @@ func NewRepoFile() *RepoFile {
 func LoadRepositoriesFile(path string) (*RepoFile, error) {
 	b, err := ioutil.ReadFile(path)
 	if err != nil {
+		if os.IsNotExist(err) {
+			fmt.Printf("Couldn't load repositories file (%s).\n"+
```

Comment:
Oh sorry, NOW I see what you mean. We should be returning `nil, fmt.Errorf()` instead or displaying on stdout. That way clients using `LoadRepositoriesFile` can have control whether or not this error message should be displayed on stdout or not. THEN we can check in pkg/repo/repo_test.go for the error message.

Reply:
Aah ok. Initial attempt is up.